TestAnimals/UnitTest1.cs

```diff
@@ -0,0 +1,45 @@
+using AnimalsTest;
+
+namespace TestAnimals
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void VoiceLion()
+        {
+            var lion = new Lion();
```

Comment:
using `var` may not be the best practice. Try to avoid it (there is an editor config rule for that.
```suggestion
            Lion lion = new();
```